frontend/javascripts/oxalis/constants.js

```diff
@@ -65,23 +65,45 @@ export const OrthoViewIndices = {
   TDView: OrthoViewValues.indexOf("TDView"),
 };
 export const OrthoViewValuesWithoutTDView = [
-  OrthoViews.PLANE_XY,
-  OrthoViews.PLANE_YZ,
-  OrthoViews.PLANE_XZ,
+  OrthoViews.PLANE_XY.id,
+  OrthoViews.PLANE_YZ.id,
+  OrthoViews.PLANE_XZ.id,
 ];
 
 export const OrthoViewColors: OrthoViewMap<number> = {
-  [OrthoViews.PLANE_XY]: 0xc81414, // red
-  [OrthoViews.PLANE_YZ]: 0x1414c8, // blue
-  [OrthoViews.PLANE_XZ]: 0x14c814, // green
-  [OrthoViews.TDView]: 0xffffff,
+  [OrthoViews.PLANE_XY.id]: 0xc81414, // red
+  [OrthoViews.PLANE_YZ.id]: 0x1414c8, // blue
+  [OrthoViews.PLANE_XZ.id]: 0x14c814, // green
+  [OrthoViews.TDView.id]: 0xffffff,
 };
 
 export const OrthoViewCrosshairColors: OrthoViewMap<[number, number]> = {
-  [OrthoViews.PLANE_XY]: [0x0000ff, 0x00ff00],
-  [OrthoViews.PLANE_YZ]: [0xff0000, 0x00ff00],
-  [OrthoViews.PLANE_XZ]: [0x0000ff, 0xff0000],
-  [OrthoViews.TDView]: [0x000000, 0x000000],
+  [OrthoViews.PLANE_XY.id]: [0x0000ff, 0x00ff00],
+  [OrthoViews.PLANE_YZ.id]: [0xff0000, 0x00ff00],
+  [OrthoViews.PLANE_XZ.id]: [0x0000ff, 0xff0000],
+  [OrthoViews.TDView.id]: [0x000000, 0x000000],
+};
+
+export const TracingTabs = {
+  DatasetInfoTabView: { id: "DatasetInfoTabView", name: "Info", shortName: "Info" },
+  TreesTabView: { id: "TreesTabView", name: "Trees", shortName: "Trees" },
+  CommentTabView: { id: "CommentTabView", name: "Comments", shortName: "Comments" },
+  MappingInfoView: { id: "MappingInfoView", name: "Volume", shortName: "Volume" },
+  MeshesView: { id: "MeshesView", name: "Meshes", shortName: "Meshes" },
+  AbstractTreeTabView: {
+    id: "AbstractTreeTabView",
+    name: "AbstractTree",
+    shortName: "AbsTree",
```

Comment:
I don't think this abbreviation is easy to understand. When would this be used?

Reply:
See my comment above.

I removed this field now as it was a bit too "eager" to add this. They are currently not used. I just remembered that we had the idea of shorter names in case of having many stacked tabs to still have as many as possible visible instead of hidden in a menu.